### PR TITLE
Replaces scheme to https for all urls returned to client side.

### PR DESCRIPTION
**Pull request by bernarden**
Fixes issue #3295.

### CODE REVIEW

File: src/Ombi.Helpers/StringHelper.cs [hunk 1]
```diff
@@ -131,7 +131,7 @@ public static string StripCharacters(this string str, params char[] chars)
 
         public static string ToHttpsUrl(this string currentUrl)
         {
-            return currentUrl.Replace("http://", "https://");
+            return currentUrl?.Replace("http://", "https://");
```
Comment (tidusjar) [inline src/Ombi.Helpers/StringHelper.cs:134]: Nice work! Looks good, 

Can we please add a unit test for this where the `currentlyUrl` is null?

**Review by tidusjar — CHANGES_REQUESTED** [review]

File: src/Ombi.Helpers/StringHelper.cs (hunk 1 above)
Comment (bernarden) [reply src/Ombi.Helpers/StringHelper.cs:134]: Done. =)

**Review by tidusjar — APPROVED** [review]
